Upgrading the Studio (#1586)
bonitasoft/bonita-doc: modules/ROOT/pages/upgradestudio.adoc

> = Upgrading your Studion

Reviewer: ```suggestion
= Upgrading your Studio
```

How to upgrade your existing Studio to a newer version.

== Upgrading to a new Minor Version

> . Read the version release notes to understand the major changes and their possible impact on your projects

Reviewer: ```suggestion
. Read the version xref:release-notes.adoc[release notes] to understand the major changes and their possible impact on your projects
```

> . Download the new Studio version

Reviewer: ```suggestion
. Download the latest Studio version
```

. Install the Studio
. Launch the Studio
. If you are using a GIT repository
 .. In your previous Studio, make sure you have pushed all the latest changes to your project
 .. Clone your project in the latest Studio version, some upgrade will require actions/decisions from your part
 .. Repeat for each project that needs to be moved to the new version
. If you aren't using a GIT repository
 .. In your previous Studio, Export your project as a BOS file
 .. Import the .BOS in the latest Studio version, some upgrade will require actions/decisions from your part
 .. Repeat for each project that needs to be moved to the new version

> == Between Maintenance version

Reviewer: ```suggestion
== Between Maintenance version (Enterprise only)
```

> Maintenance versions are cross-compatible, meaning that your Studio version must not match the target runtime version. 

Reviewer: ```suggestion
Maintenance versions are cross-compatible, meaning that it is not required that your Studio version matches the target runtime version.
```
